drivers/kw2xrf/kw2xrf.c

```diff
@@ -438,6 +439,7 @@ int kw2xrf_init(kw2xrf_t *dev, spi_t spi, spi_speed_t spi_speed,
 #if CPUID_LEN > IEEE802154_LONG_ADDRESS_LEN
     for (int i = IEEE802154_LONG_ADDRESS_LEN; i < CPUID_LEN; i++) {
         cpuid[i & 0x07] ^= cpuid[i];
+        addr_short ^= (uint16_t)(cpuid[i] << ((i & 0x01) * 8));
```

Comment:
Just for understanding of that line: you are taking every odd byte of the CPU ID and XOR it with the most-significant byte of the short address and every even byte with the least-significant byte. Is that correct? (maybe add a comment?)